laceworksdk/api/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 from dotenv import load_dotenv
 import configparser
+from laceworksdk.api.v2.inventory import InventoryAPI
```

Comment:
This shouldn't be required.
```suggestion
```

Reply:
good catch. This snuck into my commit...will remove.